python/tvm/topi/arm_cpu/conv2d_spatial_pack.py

```diff
@@ -273,7 +273,8 @@ def conv2d_spatial_pack_nhwc(cfg, data, kernel, strides, padding, dilation, out_
     data_pad = nn.pad(data, [0, pad_top, pad_left, 0], [0, pad_down, pad_right, 0])
 
     # ==================== define configuration space ====================
-    n, oc, oh, ow = cfg.axis(N), cfg.axis(OC), cfg.axis(OH), cfg.axis(OW)
+    n = cfg.axis(N) if isinstance(N, int) else cfg.axis(1)
```

Comment:
Does it mean to use `1` as the batch size when it is dynamic? Better to add a comment if so.

Reply:
Comment added. This is to set the split factor candidate to 1 when the batch size is dynamic.